File: C/Problem Set 1 - C basics/#3 Mario (less)/marioless.c
#include <cs50.h>
#include <stdio.h>

void print_row(int bricks);
void print_spaces(int spaces);

int main(void)
{
    // Ask the user about the pyramid's height
    int height;
    do
    {
        height = get_int("Pyramid's height: ");
    }
    while (height < 1);

    // Print the pyramid
    for (int i = 0; i < height; i++)
    {
        print_spaces(height - i - 1);
        print_row(i + 1);
    }
}
// Prints spaces
void print_spaces(int spaces)
{
    for (int i = 0; i < spaces; i++)
    {
        printf(" ");
    }
}

// Prints hashes
void print_row(int bricks)
{
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
    printf("\n");
}

File: Python/#2 Mario (less)/mario-less.py
from cs50 import get_int

# Loops forever until height given is within range
while True:
    height = get_int('Height: ')
    if height in range(1, 9):
        break

# Prints spaces and blocks
for i in range(height):
    print(' ' * (height - i - 1), end='')
    print('#' * (i + 1))
